### PR TITLE
fail2ban on ubuntu 20.04 tutorial

### PR DESCRIPTION
**Pull request by RigatoniDiApplicazioni**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Beatrice Richter <beatrice.richter@hetzner.com>

### CODE REVIEW

File: tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+The jail action contains the instructions what to do with IP addresses which are matched by the filters.
+These actions can be used for all jails.
+
+Create a new file named `jail-action.conf` and implement the action as follows:
+
+```bash
+[Definition]
+actionban = iptables -A INPUT -s <ip> -j DROP
+actionunban = iptables -D INPUT -s <ip> -j DROP
+```
```
Comment (rook59) [inline tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md:102]: Why do you not use a pre-defined action like iptables or iptables-allports? Those for example also add / remove the banned IP addresses if fail2ban is started/stopped/restarted

File: tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+Without suffix the ban time default is defined in seconds. You can use suffix like 1d or 10m (which is the fail2ban default).
+
+The files where the actions are defined are within `actions.d/`
+
+### Step 2.2 - jail-action.conf
+
+The jail action contains the instructions what to do with IP addresses which are matched by the filters.
+These actions can be used for all jails.
+
+Create a new file named `jail-action.conf` and implement the action as follows:
```
Comment (rook59) [inline tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md:96]:
```suggestion
Create a new file named `jail-action.conf` in `actions.d` and implement the action as follows:
```

File: tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+```bash
+[Definition]
+actionban = iptables -A INPUT -s <ip> -j DROP
+actionunban = iptables -D INPUT -s <ip> -j DROP
+```
+
+
+### Step 2.3 - sshd.conf
```
Comment (rook59) [inline tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md:105]: This file existing and already contains a pre-defined sshd filter

File: tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/tutorial-template"
+slug: "tutorial-template"
+date: "2021-18-10"
+title: "How to install Fail2ban on Ubuntu 20.04"
+short_description: "This is a tutorial explains how to install and configure Fail2ban on Ubuntu 20.04"
+tags: ["Linux", "Install", "Ubuntu"]
```
Comment (rook59) [inline tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md:8]: I think it makes sense to add fail2ban to the tags

File: tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md (hunk 1 above)
Comment (RichterBea) [reply tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md]: i will take a look at

File: tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md (hunk 4 above)
Comment (RichterBea) [reply tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md]: done

File: tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md (hunk 1 above)
Comment (RichterBea) [reply tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md]: but would iptables-allports not also remove the ban if fail2ban is stopped?

Comment (rook59) [reply tutorials/install-and-configure-fail2ban-on-ubuntu/fail2ban-on-ubuntu2004.md]: Yes, the ban would be removed but that's what you want, isn't it? Why should be ban be persistent if fail2ban got stopped? Also with the current configuration you can't distinguish whether the IP was banned manually or by fail2ban

**Pull request closed without merging**